pkg/monitor/agent/listener1_2.go

```diff
@@ -78,3 +81,11 @@ func (ml *listenerv1_2) drainQueue() {
 func (ml *listenerv1_2) Version() listener.Version {
 	return listener.Version1_2
 }
+
+// Close closes the underlying socket and payload queue.
+func (ml *listenerv1_2) Close() {
+	ml.once.Do(func() {
+		ml.conn.Close()
+		close(ml.queue)
```

Comment:
This `close()` call is new and not mentioned in the commit message. Why is it needed?

Reply:
this is added to unblock drainQueue goroutine in unit test:

https://github.com/cilium/cilium/pull/9924#discussion_r369336250

i'll update the commit message

Reply:
@tgraf i updated the commit message